app/policies/user_policy.rb

```diff
@@ -0,0 +1,20 @@
+# app/policies/user_policy.rb
+
+class UserPolicy
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def show?
+    user == current_user ||
+     !user.private? || 
+     user.followers.include?(current_user)
+  end
+
+  def feed?
+    true
+  end
+end
```

Comment:
I think you need to add the discover and liked methods and set to true. I am getting an error when I click on "Discover" and when I click on someone's "Liked" photos

Reply:
agreed